index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>

```

Comment:
It won;t work like this. You didn't updated the index.html file inside www folder. Since you are working with ionic you should know that.